pages/_document.js

```diff
@@ -1,8 +1,13 @@
 import Document, { Html, Head, Main, NextScript } from 'next/document';
 import React from 'react';
 import GoogleFonts from 'next-google-fonts';
+import { ColorModeScript } from '@chakra-ui/core';
 
 class MyDocument extends Document {
+  static getInitialProps(ctx) {
```

Comment:
I don't think this is needed, correct?

Reply:
Probably yes, Just added from the demo provided in chakra docs